astropy/cosmology/units.py

```diff
@@ -70,36 +70,52 @@ def dimensionless_redshift():
 def redshift_distance(cosmology=None, kind="comoving", **atzkw):
     """Convert quantities between redshift and distance.
 
-    Care should be taken to not misinterpret a relativistic, gravitational, etc
-    redshift as a cosmological one.
+    Care should be taken to not misinterpret a relativistic, gravitational, etc redshift
+    as a cosmological one.
 
     Parameters
     ----------
     cosmology : `~astropy.cosmology.Cosmology`, str, or None, optional
-        A cosmology realization or built-in cosmology's name (e.g. 'Planck18').
-        If None, will use the default cosmology
-        (controlled by :class:`~astropy.cosmology.default_cosmology`).
+        A cosmology realization or built-in cosmology's name (e.g. 'Planck18'). If None,
+        will use the default cosmology (controlled by
+        :class:`~astropy.cosmology.default_cosmology`).
     kind : {'comoving', 'lookback', 'luminosity'}, optional
-        The distance type for the Equivalency.
-        Note this does NOT include the angular diameter distance as this
-        distance measure is not monotonic.
+        The distance type for the Equivalency. Note this does NOT include the angular
+        diameter distance as this distance measure is not monotonic.
     **atzkw
-        keyword arguments for :func:`~astropy.cosmology.z_at_value`
+        keyword arguments for :func:`~astropy.cosmology.z_at_value`, which is used to
+        convert distance to redshift.
 
     Returns
     -------
     `~astropy.units.equivalencies.Equivalency`
         Equivalency between redshift and temperature.
 
+    Raises
+    ------
+    `astropy.comology.CosmologyError`
```

Comment:
```suggestion
    `~astropy.comology.CosmologyError`
```

Reply:
Thanks. `cosmology` was also mispelled, leading to RTD failure.